code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -588,3 +588,75 @@
 
 /obj/effect/mob_spawn/human/pirate/gunner
 	rank = "Gunner"
+
+//Forgotten syndicate ship
+
+/obj/effect/mob_spawn/human/syndicatespace
+	name = "Syndicate Ship Crew Member"
+	roundstart = FALSE
+	death = FALSE
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper_s"
+	short_desc = "You are a syndicate operative on old ship, stuck in hostile space."
+	flavour_text = "Your ship docks after a long time somewhere in hostile space, reporting a malfunction. You are stuck here, with Nanotrasen station nearby. Fix the ship, find a way to power it and follow your captain's orders."
+	important_info = "Obey orders given by your captain. DO NOT leave the ship."
+	outfit = /datum/outfit/syndicatespace/syndicrew
+
+/datum/outfit/syndicatespace/syndicrew/post_equip(mob/living/carbon/human/H)
+	H.faction |= ROLE_SYNDICATE
+
+/obj/effect/mob_spawn/human/syndicatespace/special(mob/living/new_spawn)
+	new_spawn.grant_language(/datum/language/codespeak, TRUE, TRUE, LANGUAGE_MIND)
+
+/obj/effect/mob_spawn/human/syndicatespace/Destroy()
+	new/obj/structure/fluff/empty_sleeper/syndicate(get_turf(src))
+	..()
+
+
+/obj/effect/mob_spawn/human/syndicatespace/captain
+	name = "Syndicate Ship Captain"
+	roundstart = FALSE
+	death = FALSE
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper_s"
+	short_desc = "You are the captain of an old ship, stuck in hostile space."
+	flavour_text = "Your ship docks after a long time somewhere in hostile space, reporting a malfunction. You are stuck here, with Nanotrasen station nearby. Command your crew and turn your ship into the most protected fortress."
```

Comment:
Is the captain allowed to raid the vault for the cool hardsuits himself?

Is he allowed to add the password on the paper to his notes menu, then burn it so that no one else can read it?

Reply:
Opening the vault is a point of no-return, you can't lock it once you opened it.
The vault itself is made to store secure loot while there is no active captain.

Reply:
> Maybe you should add a GPS signal to this ruin (that'll only appear if the captain spawns, sort of like the comms base GPS signal?) to increase the likelyhood that the syndicate crew members will have someone to fight?

I was thinking of it, but the best thing I could think of is giving the captain with tracking implant on spawn, without teleportation part.